src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -365,7 +365,7 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
             if (this.ShouldLogEvent(isReplay: false))
             {
                 this.logger.LogInformation(
-                    "{instanceId}: Function '{functionName} ({functionType})' dropped a '{eventName}' event. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
+                    "{instanceId}: Function '{functionName} ({functionType})' saved a '{eventName}' event to an in-memory queue. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
                     instanceId, functionName, functionType, eventName, FunctionState.ExternalEventDropped, hubName,
```

Comment:
@cgillum Chris, I am wondering why we have the FunctionState named as `ExeternalEventDropped`. 

I am asking because I had GH issue and also IcM tickets regrading this. Customers thought the external event was dropped/ not processed. But actually, it's a normal behavior and the event was saved to the in-memory buffer.  Just the naming is misleading.  Also, considering this is the only scenario that this FunctionState will be called. Do you think it would be better for us to update the FunctionState as `ExeternalEventSaved`?

Reply:
Thanks for the reply, @cgillum !

What I want to update is [here](https://github.com/Azure/azure-functions-durable-extension/blob/5453308c9394b7f33633912802a36c8c0e577809/src/WebJobs.Extensions.DurableTask/FunctionState.cs#L16) to "ExternalEventSaved". We did rename the TaskName as "ExternalEventSaved", which is seen at Kusto logs. Customers can still see function state as "ExternalEventDropped" like this [post](https://github.com/Azure/azure-functions-durable-extension/issues/2482#issuecomment-1584178881). 

I have a ticket again today about this, so I am asking if we should update the FunctionState enum item name. Let me know if any is unclear to you. Thank you!